akka-cluster-sharding-typed/src/multi-jvm/scala/akka/cluster/sharding/typed/ReplicatedShardingSpec.scala

```diff
@@ -0,0 +1,189 @@
+/*
+ * Copyright (C) 2020 Lightbend Inc. <https://www.lightbend.com>
+ */
+
+package akka.cluster.sharding.typed
+
+import akka.actor.typed.scaladsl.LoggerOps
+import akka.Done
+import akka.actor.testkit.typed.scaladsl.TestProbe
+import akka.actor.typed.ActorRef
+import akka.actor.typed.scaladsl.ActorContext
+import akka.actor.typed.scaladsl.Behaviors
+import akka.cluster.MultiNodeClusterSpec
+import akka.cluster.sharding.typed.ReplicatedShardingSpec.TestRES.GetState
+import akka.cluster.sharding.typed.ReplicatedShardingSpec.TestRES.State
+import akka.cluster.sharding.typed.ReplicatedShardingSpec.TestRES.StoreMe
+import akka.cluster.sharding.typed.scaladsl.Entity
+import akka.cluster.typed.MultiNodeTypedClusterSpec
+import akka.persistence.journal.PersistencePluginProxy
+import akka.persistence.testkit.query.scaladsl.PersistenceTestKitReadJournal
+import akka.persistence.typed.ReplicaId
+import akka.persistence.typed.ReplicationId
+import akka.persistence.typed.scaladsl.ReplicatedEventSourcing
+import akka.persistence.typed.scaladsl.Effect
+import akka.persistence.typed.scaladsl.EventSourcedBehavior
+import akka.remote.testkit.MultiNodeConfig
+import akka.remote.testkit.MultiNodeSpec
+import akka.serialization.jackson.CborSerializable
+import com.typesafe.config.ConfigFactory
+import org.scalatest.concurrent.Eventually
+import org.scalatest.concurrent.ScalaFutures
+import org.scalatest.time.Span
+
+object ReplicatedShardingSpec extends MultiNodeConfig {
+  val first = role("first")
+  val second = role("second")
+
+  commonConfig(ConfigFactory.parseString("""
+    akka.loglevel = DEBUG
+    akka.persistence.journal.plugin = "akka.persistence.journal.inmem"
+    // for the proxy plugin
+    akka.actor.allow-java-serialization = on 
```

Comment:
also `akka.actor.warn-about-java-serializer-usage = off`

Reply:
we should maybe add a serializer to the testkit as i nearly missed the other missing serializer because of this